src/site/content/en/media/video-and-source-tags/index.md

```diff
@@ -0,0 +1,336 @@
+MPEG-4 videos.
+
+This approach has several advantages over serving different HTML or server-side
+scripting, especially on mobile:
+
+* You can list formats in order of preference.
+* Native client-side switching reduces latency; only one request is made to
+  get content.
+* Letting the browser choose a format is simpler, quicker, and potentially
+  more reliable than using a server-side support database with user-agent detection.
+* Specifying each file source's type improves network performance; the browser can select a
+  video source without having to download part of the video to "sniff" the format.
+
+These issues are especially important in mobile contexts, where bandwidth and
+latency are at a premium and the user's patience is likely limited. Omitting the
+type attribute can affect performance when there are multiple sources with
+unsupported types.
+
+{% Aside %} There are a few ways you can dig into the details. Check out [A
+Digital Media Primer for Geeks](//www.xiph.org/video/vid1.shtml) to find out
+more about how video and audio work on the web. You can also use [remote
+debugging](https://developers.google.com/web/tools/chrome-devtools/remote-debugging)
+in DevTools to compare network activity [with type
+attributes](https://googlesamples.github.io/web-fundamentals/fundamentals/media/video-main.html)
+and [without type
+attributes](https://googlesamples.github.io/web-fundamentals/fundamentals/design-and-ux/responsive/notype.html).
+
+Also check the response headers in your browser developer tools to
+[ensure your server reports the right MIME type](//developer.mozilla.org/en/docs/Properly_Configuring_Server_MIME_Types);
+otherwise video source type checks won't work.
+{% endAside %}
+
+### Specify start and end times
+
+Save bandwidth and make your site feel more responsive: use media fragments to
+add start and end times to the video element.
+
+<video controls>
+  <source src="https://storage.googleapis.com/web-dev-assets/video-and-source-tags/chrome.webm#t=5,10" type="video/webm">
+  <source src="https://storage.googleapis.com/web-dev-assets/video-and-source-tags/chrome.mp4#t=5,10" type="video/mp4">
+  <p>This browser does not support the video element.</p>
+</video>
+
+To use a media fragment, add `#t=[start_time][,end_time]` to the media URL. For
+example, to play the video from seconds 5 to 10, specify:
+
+```html
+<source src="video/chrome.webm#t=5,10" type="video/webm">
+```
+
+You can use this feature to deliver multiple views on the same video&ndash;like
+cue points in a DVD&ndash;without having to encode and serve multiple files.
+
+{% Aside 'caution' %}
+By default, most servers enable range requests, which is required for media
+fragments to work. Because some hosting services turn them off, you should test
+this feature with your server before using it on a live site.
+{% endAside %}
+
+Using your browser developer tools, check for `Accept-Ranges: bytes` in the
```

Comment:
I found the context for this confusing. Is this to determine whether ranges are supported?

Reply:
This is related to the previous caution. I started to rework these two sections and the image into a range requests section. The problem is, it's not used anywhere else and as far as I know it's not relevant to videos in &lt;video> tags. What do you think about removing it?

Reply:
Remove the range section completely? I think it's useful to have. It just wasn't clear to me that the paragraph was related to the caution.